scripts/publish_api_coverage.py

```diff
@@ -164,13 +164,20 @@ def generate_pandas_api_coverage():
                 token_type = "property"
 
             is_in_bigframes = hasattr(bigframes_obj, member)
-            requires_index = False
-            requires_ordering = False
+            requires_index = ""
```

Comment:
Nit: I guess changing it from bool to "" and "Y" for type consistency? But bool seems more readable.

Reply:
I'd like to support "partial" in future, so using string as a stand-in for an enum.